core/store/migrate/migrations/0158_update_job_proposal_status.sql

```diff
@@ -0,0 +1,93 @@
+-- +goose Up
+-- +goose StatementBegin
+-- JobProposals Table
+-- We must remove the old contraint to add an enum value to support Postgres v11
+ALTER TABLE job_proposals DROP CONSTRAINT chk_job_proposals_status_fsm;
```

Comment:
You can add new enum types since Postgres 9.1 without needing to create a new type.

Reply:
response got lost in the long history; Unfortunately, this won't allow me to update the null constraint check in the same migration as I get Unsafe use of new value of enum type error for trying to use the new constraint in the same transaction.